test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -0,0 +1,356 @@
+# SPDX-License-Identifier: Apache-2.0
+
+set(ONNX_BACKENDTEST_SRC_DIR ${ONNX_MLIR_SRC_ROOT}/test/backend)
+set(ONNX_BACKENDTEST_BIN_DIR ${ONNX_MLIR_BIN_ROOT}/test/backend)
+
+file(GENERATE
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/conftest.py
+  INPUT ${ONNX_BACKENDTEST_SRC_DIR}/conftest.py
+  )
+
+file(GENERATE
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/test.py
+  INPUT ${ONNX_BACKENDTEST_SRC_DIR}/test.py
+  )
+
+file(GENERATE
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/inference_backend.py
+  INPUT ${ONNX_BACKENDTEST_SRC_DIR}/inference_backend.py
+  )
+
+file(GENERATE
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/signature_backend.py
+  INPUT ${ONNX_BACKENDTEST_SRC_DIR}/signature_backend.py
+  )
+
+file(GENERATE
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/variables.py
+  INPUT ${ONNX_BACKENDTEST_SRC_DIR}/variables.py
+  )
+
+file(GENERATE
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/common.py
+  INPUT ${ONNX_BACKENDTEST_SRC_DIR}/common.py
+  )
+
+configure_file(
+ ${ONNX_BACKENDTEST_SRC_DIR}/test_config.py.in
+ ${ONNX_BACKENDTEST_BIN_DIR}/test_config.py.cfg
+ @ONLY
+ )
+
+file(GENERATE
+ OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/test_config.py
+ INPUT ${ONNX_BACKENDTEST_BIN_DIR}/test_config.py.cfg
+ )
+
+configure_file(
+  ${ONNX_BACKENDTEST_SRC_DIR}/test_config_compilerlib.py.in
+  ${ONNX_BACKENDTEST_BIN_DIR}/test_config_compilerlib.py.cfg
+  @ONLY
+  )
+
+file(GENERATE
+ OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/test_config_compilerlib.py
+ INPUT ${ONNX_BACKENDTEST_BIN_DIR}/test_config_compilerlib.py.cfg
+ )
+
+# CMAKE_CFG_INTDIR is . for single-config generators such as make, and
+# it has a value (e.g. $(Configuration)) otherwise, so we can use it to
+# determine whether we are dealing with a multi-config generator.
+if (NOT "${CMAKE_CFG_INTDIR}" STREQUAL ".")
+  set(FILE_GENERATE_DIR ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_CFG_INTDIR})
+else()
+  set(FILE_GENERATE_DIR ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE})
+endif()
+
+# Detect pytest-xdist for parallel backend tests
+execute_process(
+  COMMAND ${Python3_EXECUTABLE} -m pip show pytest-xdist
+  RESULT_VARIABLE PYTEST_XDIST_FOUND
+  OUTPUT_QUIET
+  ERROR_QUIET
+)
+if (${PYTEST_XDIST_FOUND} EQUAL 0)
+  message(STATUS "Parallel backend tests   : ON")
+  set(BACKEND_TEST_COMMAND "${Python3_EXECUTABLE}" "-m" "pytest")
+  set(BACKEND_TEST_ARGS "--forked" "-n" "$$\{NPROC:-auto\}" "-q" "--silent")
+else()
+  message(STATUS "Parallel backend tests   : OFF (install pytest-xdist to enable)")
+  set(BACKEND_TEST_COMMAND ${Python3_EXECUTABLE})
+  set(BACKEND_TEST_ARGS "")
+endif()
+
+# Followings are test cases of test_to_enable_dict in test/backend/inference_backend.py.
+# Only test cases supported by zDNN are listed.
```

Comment:
Do you mind explaining the format? is it paris of <backend-test> <zdnn-op-being-testted>

Reply:
Thanks for your comment. I added a sentence to explain. Also I updated [AccelNNPAHowToUseAndTest.md](https://github.com/imaihal/onnx-mlir/blob/nnpa_backendtest/docs/AccelNNPAHowToUseAndTest.md) and [Testing.md](https://github.com/imaihal/onnx-mlir/blob/nnpa_backendtest/docs/Testing.md) to explain the environment variables.